src/components/header/header.js

```diff
@@ -0,0 +1,21 @@
+import './header.css';
+
+const Header = () => (
+  <header>
+    <div className="cont-nav">
+      <h2 className="navbar-title">Bookstore</h2>
+      <nav>
+        <ul className="navbar-list">
+          <li className="navbar-option">
+            <a className="navbar-link" href="/">Books</a>
+          </li>
+          <li className="navbar-option">
+            <a className="navbar-link" href="/categories">Categories</a>
+          </li>
+        </ul>
```

Comment:
- [ ] Well done implementing the links 👍🏼, they work just fine. It would be nice to see if we'll achieve the same results if we used the recommended way as per the project's requirements. Look at the requirement below:

---
> Add React Router and set two `<Route>s` and `<Link>s` for the app's navigation
---

Instead of using our own `<a>` tag, the above requirement expects us to use the `<Link>` component that comes bundled with the react-router-dom package. Implementing our own anchor tag defeats the purpose of installing the react-router-dom package, it becomes a case of reinventing the wheel.

Please import react-router-dom into this file and refactor all `<a>` tags to `<Link>` component from react-router-dom.